### ISSUE TITLE
Issue with latest Craft / Commerce - needs Guzzle 7.

### ISSUE THREAD

**Issue by bossanova808** [matt-west/craft-recaptcha#23]
It seems all the payment gateways for Commerce have moved to requiring Guzzle 7, which means those now conflict with this plugin:

```
  Problem 1
    - Root composer.json requires craftcms/commerce-eway ^3.0.0 -> satisfiable by craftcms/commerce-eway[3.0.0].
    - Conclusion: don't install craftcms/cms[3.6.11.2] | install guzzlehttp/guzzle[7.2.0] (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.2.0 (conflict analysis result)
    - Conclusion: don't install guzzlehttp/guzzle 7.3.0 (conflict analysis result)
    - Conclusion: don't install craftcms/cms 3.6.12 (conflict analysis result)
    - Root composer.json requires matt-west/craft-recaptcha ^1.0 -> satisfiable by matt-west/craft-recaptcha[1.0.0, ..., 1.5.1].
    - craftcms/commerce-eway 3.0.0 requires craftcms/cms ^3.6 -> satisfiable by craftcms/cms[3.6.0-beta.1, ..., 3.6.12].
    - craftcms/cms[3.6.0-beta.1, ..., 3.6.11.1] require guzzlehttp/guzzle ^6.5.5|^7.2.0 -> satisfiable by guzzlehttp/guzzle[6.5.5, 7.2.0, 7.3.0].
    - Conclusion: don't install guzzlehttp/guzzle 6.5.5 (conflict analysis result)
```

I don't immediately need to upgrade those, as they're just dependency changes and not functionality ones, it seems...but this is a sign of the future I think and I am hoping you'd be willing to update this one to be guzzle 7 friendly?

**Comment by bob-pixeldeluxe** [comment]
We are also experiencing this issue.

**Comment by studio-jw** [comment]
Any updates on this? Same issue here.

**Comment by robinbraibant** [comment]
Same issue over here. Any updates concerning this problem?

**Comment by matt-west** [comment]
Hey folks! @bossanova808 kindly provided a fix for this that has been released in v1.5.2
